app/services/remove_cps_service.rb

```diff
@@ -0,0 +1,58 @@
+class RemoveCpsService
+  def call
```

Comment:
Can this whole method be wrapped in a transaction?

Reply:
good point

Reply:
wrapped it in a transaction